CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
 #============================================================================
 # Configure the project
 #============================================================================
-ign_configure_project(VERSION_SUFFIX pre1)
+ign_configure_project(VERSION_SUFFIX)
```

Comment:
We should keep `main` at `pre1`

Reply:
fixed in a force-push https://github.com/ignitionrobotics/ign-fuel-tools/pull/241/commits/bab9bb1ddb0136200d5dc5e6d6d4183e9fcb3231